bcipy/helpers/triggers.py

```diff
@@ -651,7 +651,10 @@ class TriggerType(Enum):
     TARGET = "target"
     FIXATION = "fixation"
     PROMPT = "prompt"
-    SYSTEM = "offset"
+    SYSTEM = "offset_correction"
```

Comment:
I have a PR to change how we write and read system offsets. It will be as implemented (with a single offset written). I think it'd be best to wait for that PR. I should be done this week.

Reply:
Okay, sounds good. If possible, I would like to support any previously used trigger types.

Reply:
We discussed this a bit in the Trigger PR but the implications are pretty far reaching. As of 2.0, we discussed dropping support for these older offset correction methods. If requested, we could provide a helper to convert older data to this newer format. This would require reading in the triggers.txt and writing a single offset value to the triggers file.